fe-public/src/components/Scoreboard.astro

```diff
@@ -0,0 +1,130 @@
+---
+import getSortedPosts from "../scripts/load-posts";
+
+let averageCommitContent = 0;
+
+let users = [
```

Comment:
users should be fetched into a variable / not hard coded.

Reply:
Im assuming this will be resolved with resolving the MR conflict

Reply:
They are, it's just the old code that somehow made it into this PR.